### DISCUSSION TITLE
Event Weighted Ranking Points calculation

### DISCUSSION THREAD

**Discussion by aivoprykk** [djong1/gp3s#48] [Q&A] [answered]
How are these numbers calculated?

**Comment by djong1** [comment, marked as answer]
For every category we create a ranking, based on the listing in the ranking the rider get's a value assigned that corresponds with his listing. So a seventh place corresponds with 7 points. This will then be combined for all category's ending in a total score. The avg standing over all disciplines is also added to the total score, to eliminate the effect of 1 lucky run that gave you a great 100, 250 and or 500 meter.
